kuma/javascript/src/document-provider.jsx

```diff
@@ -80,9 +80,20 @@ export default function DocumentProvider(
                     }
                 })
                 .then(json => {
-                    window.scrollTo(0, 0);
-                    setDocumentData(json);
-                    body.style.opacity = '1';
+                    if (json.redirectURL) {
+                        window.location = json.redirectURL;
+                    } else {
+                        let jsonLocaleAndSlug = `/${json.locale}/${json.slug}`;
+                        if (jsonLocaleAndSlug !== localeAndSlug) {
+                            // This was a redirect.
+                            let jsonURL = json.absoluteURL;
```

Comment:
I'd either rename this to receivedURL, or just drop the variable and use json.absoluteURL in the two places below.